generate/templates/couchbase-edge-server/Dockerfile.template

```diff
@@ -0,0 +1,40 @@
+FROM {{ .DOCKER_BASE_IMAGE }}
+
+LABEL maintainer="docker@couchbase.com"
+
+ENV PATH $PATH:/opt/couchbase-edge-server/bin
+
+# Install dependencies:
+RUN set -x \
+    && apt update \
+    && apt install -y \
+           curl \
+           lsb-release \
+           systemctl \
+           wget \
+           zlib1g \
+    && apt clean
+
+RUN groupadd -g 1000 couchbase && useradd couchbase -u 1000 -g couchbase -M
+
+# Install Couchbase-Edge-Server
+ARG EDGE_SERVER_RELEASE_URL="{{ .CB_RELEASE_URL }}"
+ARG EDGE_SERVER_PACKAGE_FILENAME="{{ .CB_PACKAGE_NAME }}"
+RUN set -x \
+    && EDGE_SERVER_PACKAGE_FILENAME=$(echo ${EDGE_SERVER_PACKAGE_FILENAME} | sed -e "s/@@ARCH@@/$(dpkg --print-architecture)/") \
+    && wget ${EDGE_SERVER_RELEASE_URL}/${EDGE_SERVER_PACKAGE_FILENAME} \
+    && apt install -y ./${EDGE_SERVER_PACKAGE_FILENAME} \
+    && rm ${EDGE_SERVER_PACKAGE_FILENAME} \
+    && rm -f /usr/lib/systemd/system/couchbase-edge-server.service \
+    && apt autoremove \
+    && apt clean
+
+# Add bootstrap script
+COPY scripts/entrypoint.sh /
+ENTRYPOINT ["/entrypoint.sh"]
+CMD ["/opt/couchbase-edge-server/etc/config.json"]
+
+USER couchbase
+WORKDIR /opt/couchbase-edge-server
```

Comment:
For potential interactive use of the `couchbase-edge-server --add-user` command, it'd be nice if `WORKDIR` was set to `/opt/couchbase-edge-server/etc`.

Reply:
changed.